tp_tests.ml

```diff
@@ -23,6 +23,9 @@
    {test = "`(a b c)"; expected_result = "(cons 'a (cons 'b (cons 'c '())))"};
    {test = "`(,a b ,c)"; expected_result = "(cons a (cons 'b (cons c '())))"};
    {test = "`(,@a b ,@c)"; expected_result = "(append a (cons 'b c))"};
+   {test = "(or expr1 expr2)"; expected_result = ScmOr [ScmVarGet (Var "expr1"); ScmVarGet (Var "expr2")]};
+   {test = "(or)"; expected_result = ScmConst (ScmBoolean false)};
+   {test = "(or expr)"; expected_result = ScmVarGet (Var "expr")};
```

Comment:
Format of expected_result fields don't match - need to be a string produced as explained above